### ISSUE TITLE
Prepare clean table for modelling

### ISSUE THREAD

**Issue by tsufz** [tsufz/mar23cds_accidents#27]
- In relation to the agreements in the data audit template:
- [x] Drop all variables set to `delete`
- [x] Replace the missing values (`NAN`) in variables having an encoded "unknown" category (`-1')
- [x] Save data to a new file
- [x] Push the file to the repository
- [x] Update the correlation matrix (-> Tobias)

- The target variable is `fatal`.
- The data can be split to training set (2015-2019) and test set (2020-2021) in the modelling notebooks.

**Comment by JohannaS111** [comment]
Hey @tsufz , i took a look on the template again regarding the point **"Impute other missing values by most frequent values"**... I guess, we didn't identify any column for doing that... or am i wrong? 🧐

**Comment by tsufz** [comment]
@JohannaS111, I checked the script. Indeed, we don't have to impute anything. Regarding the datetime values, exporting to csv will not preserve this. Hence, it is required to rerun the setting to datatime always after loading a csv.

Best,
Tobias
